src/main/java/teammates/ui/webapi/CreateFeedbackSessionLogAction.java

```diff
@@ -17,7 +25,45 @@ void checkSpecificAccessControl() {
 
     @Override
     JsonResult execute() {
-        // TODO: implement the logic for creating a feedback session log
+        String courseId = getNonNullRequestParamValue(Const.ParamsNames.COURSE_ID);
+        String fsName = getNonNullRequestParamValue(Const.ParamsNames.FEEDBACK_SESSION_NAME);
+        FeedbackSessionAttributes feedbackSession = getNonNullFeedbackSession(fsName, courseId);
+        if (feedbackSession == null) {
+            return new JsonResult("No feedback session with name " + fsName
+                    + " for course " + courseId, HttpStatus.SC_NOT_FOUND);
+        }
+
+        String fslType = getNonNullRequestParamValue(Const.ParamsNames.FEEDBACK_SESSION_LOG_TYPE);
+        if (!fslType.equals(FeedbackSessionLogTypes.ACCESS) && !fslType.equals(FeedbackSessionLogTypes.SUBMISSION)) {
+            return new JsonResult("Invalid feedback session log type.", HttpStatus.SC_BAD_REQUEST);
+        }
+
+        StudentAttributes student;
+        String studentEmail = getRequestParamValue(Const.ParamsNames.STUDENT_EMAIL);
```

Comment:
Isn't the student email non null too?

Reply:
We can make student email param non null -- but just to double-check, will the frontend have no problems getting the student email, regardless of whether the student is registered and logged-in or not?

Reply:
Yes, the frontend should have no problems sending this email back